test/e2e_kubeadm/BUILD

```diff
@@ -19,6 +19,7 @@ go_test(
         "kubelet_config_test.go",
         "nodes_test.go",
         "proxy_addon_test.go",
+        "networking_test.go",
```

Comment:
it seems the job `pull-kubernetes-verify` is falling because of this 🤔

Reply:
yes. bad ordering. i'll fix it.